hrm-service/service-tests/permissions.test.js

```diff
@@ -0,0 +1,68 @@
+const supertest = require('supertest');
+const each = require('jest-each').default;
+const expressApp = require('../src/app');
+import { rulesMap } from '../src/permissions';
+
+const server = expressApp.listen();
+const request = supertest.agent(server);
+
+const headers = {
+  'Content-Type': 'application/json',
+  Authorization: `Basic ${Buffer.from(process.env.API_KEY).toString('base64')}`,
+};
+
+afterAll(done => {
+  server.close(() => {
+    done();
+  });
+});
+
+describe('/permissions route', () => {
+  describe('GET', () => {
+    each([
+      {
+        headersConfig: {},
+        accountSid: 'notConfigured',
+        description: 'Should return status 401 (Authorization failed)',
+        expectedStatus: 401,
+        expectedPayload: 'Authorization failed',
+      },
+      {
+        accountSid: 'notConfigured',
+        description: 'Should return status 500 (permissions env var set to empty value)',
+        expectedStatus: 500,
+        expectedPayload: 'No permissions set for account notConfigured.',
+      },
+      {
+        accountSid: 'missingInEnvVars',
```

Comment:
Should some of these cases throw client errors rather than 500? This one seems more like a 404 or a 400.

Reply:
@stephenhand Sorry I've merged this PR, I plan to address this if needed in my current task.